lib/ddtrace/span.rb

```diff
@@ -31,6 +31,9 @@ class Span
     # parsing 64-bit integers for distributed tracing since an upstream system may generate one
     EXTERNAL_MAX_ID = 2**64
 
+    # This limit is for numeric tags because uint64 could end up rounded.
+    NUMERIC_TAG_SIZE_RANGE = ((-1 * 2**53)..2**53)
```

Comment:
Super minor, but I think this can be simplified like so:
```suggestion
    NUMERIC_TAG_SIZE_RANGE = (-2**53..2**53)
```